DeveloperConsole.cs

```diff
@@ -100,6 +116,7 @@ private void Initialize()
             if (!isInGame) return;
             initialized = true;
             Player = LC_Lib.GetPlayer("Player");
+            Terminal = LC_Lib.GetTerminal();
```

Comment:
GetTerminal method does not exist on LC_Lib

Reply:
This should be fixed due to the push to the Library. Mod-Library now includes a "GetTerminal" method